recipes-kernel/linux/linux-hikey-aosp_4.4.bb

```diff
@@ -71,4 +71,6 @@ do_configure() {
     bbplain "Saving defconfig to:\n${B}/defconfig"
     oe_runmake -C ${B} savedefconfig
     cp -a ${B}/defconfig ${DEPLOY_DIR_IMAGE}
+    bbplain "Saving .config to:\n${B}/config"
+    cp -a ${B}/.config ${DEPLOY_DIR_IMAGE}/config
```

Comment:
And while building OE image, We are adding kselftest required kernel config fragments.
Full config is very much useful in case of failure investigation.